www/css/main.css

```diff
@@ -1057,10 +1057,33 @@ div.running-text{
 .noweather #clock-s {
     padding-top:0;
 }
-.debugWU tr td:first-child {
+.debugWUHeading {
+    text-align:center;
+    font-weight:bold;
+}
+.debugWUHeading:not(:first-of-type) {
+    margin-top: 8px;
+    border-top: thin inset  #f3f3f3;
+    padding-top: 8px;
+}
+.debugWUTable tr td:first-child {
+    vertical-align: top;
     width: 160px;
     font-weight: 700;
 }
+.debugWUScrollable {
+    max-height: 150px;
+    overflow: auto;
+}
+.debugWUError {
```

Comment:
Since these are the same color, can we not use the same class instead of defining them twice?

Reply:
Yeah, I was playing around with red|amber|green to differentiate between offline (red), online (green) and online but with errors (amber). But for the user, I dont think there is a big difference in "colour|severity" between cant connect (offline) and bad API key (online with errors) ! So I'll consolidate error and warning.